### PR TITLE
Add searchResults pages example

### PR DESCRIPTION
**Pull request by NzingaHolloman**

### CODE REVIEW

File: src/main/webapp/brazilSearchResults.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
```
Comment (josueetcom) [inline src/main/webapp/brazilSearchResults.html:2]: **zxx** doesn't seem to be valid language code according to [this](https://www.w3schools.com/tags/att_global_lang.asp)

This shows up in the other files too

File: src/main/webapp/styleNzinga.css [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+div {
+  margin: 0;
+}
+
+/* Add a black background color to the top navigation bar */
+.topnav {
+  overflow: hidden;
+  background-color: #e9e9e9;
+  text-align: center;
```
Comment (josueetcom) [inline src/main/webapp/styleNzinga.css:24]: [Style](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order): Alphabetize declarations.

Here and elsewhere

**Review by josueetcom — APPROVED** [review]
Just two minor comments that could be addressed later when we merge the HTML/CSS for this

File: src/main/webapp/koreaSearchResults.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+    <h2>Cities to Explore</h2>
+    <div class="row">
+      <div class="column">
+        <div class="card">
+          <h1 text-align="left">Seoul</h1>
+          <img src="krPhotos/krCitySeoul.jpg" height="300" alt="South Korea city " width=" 100%;" />
+          <p class="title">
+            <!-- Seoul, the capital of South Korea, is a huge metropolis where modern skyscrapers, high-tech subways and pop
+            culture meet Buddhist temples, palaces and street markets. -->
```
Comment (gferioli0418) [inline src/main/webapp/koreaSearchResults.html:34]: Is this supposed to be a comment?

File: src/main/webapp/styleNzinga.css [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+  color: #00bcd4;
+}
+
+/* Style the search box inside the navigation bar */
+.topnav input[type='text'] {
+  float: right;
+  padding: 6px;
+  border: none;
+  margin-top: 8px;
+  margin-right: 16px;
```
Comment (gferioli0418) [inline src/main/webapp/styleNzinga.css:54]: consider using shorthand [properties](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)

File: src/main/webapp/styleNzinga.css (hunk 2 above)
Comment (NzingaHolloman) [reply src/main/webapp/styleNzinga.css:24]: I'll fix in a later pr

File: src/main/webapp/brazilSearchResults.html (hunk 1 above)
Comment (NzingaHolloman) [reply src/main/webapp/brazilSearchResults.html:2]: I'll fix in a later pr

File: src/main/webapp/koreaSearchResults.html (hunk 3 above)
Comment (NzingaHolloman) [reply src/main/webapp/koreaSearchResults.html:34]: I'll fix in a later pr

File: src/main/webapp/styleNzinga.css (hunk 4 above)
Comment (NzingaHolloman) [reply src/main/webapp/styleNzinga.css:54]: I'll fix in a later pr

**Pull request merged**